apps/comments-ui/src/components/ContentBox.tsx

```
@@ -46,7 +46,21 @@ const ContentBox: React.FC<Props> = ({done}) => {
         }
     };
 
-    const containerClass = darkMode() ? 'dark' : '';
+    useEffect(() => {
+        const el = document.getElementById(ROOT_DIV_ID);
+        if (el && el.parentElement) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)